### PR TITLE
Bookstorestyling

### PR DESCRIPTION
**Pull request by mumbereh**
### _**In this milestone I Implemented the following**_

- Added any missing JSX elements to your project (without adding extra functionality)
- The final design should match Zeplin's design (this includes all elements + the same styling)
- I applied  React best practices and language style guides in code.

### CODE REVIEW

File: src/redux/book/booksSlice.js [hunk 1]
```diff
@@ -1,52 +1,97 @@
...
+// Async Thunks
+const fetchBooks = createAsyncThunk('books/fetchBooks', async () => {
+  const response = await axios.get(`${baseApiUrl}books`);
+  return response.data;
+});
+
+const addBook = createAsyncThunk('books/addBook', async (book) => {
+  const response = await axios.post(`${baseApiUrl}books`, book);
+  return response.data === 'Created' ? book : null;
+});
```
Comment (Johnadibe) [inline src/redux/book/booksSlice.js:14]: > - [ ] Hello, I've noticed that when I try to add a book and check if its design matches the provided design, the book doesn't appear in the user interface or the Redux store. Could you please look into this matter to improve the overall quality of your application? 👍 Additionally, please keep in mind that when resolving this issue, it should adhere to the provided design requirements. 😊
> 
> This is the video to illustrate what I mean:
> 
> [screen-capture (1).webm](https://github.com/mumbereh/bookstore/assets/40009354/5d04c130-0147-4cf5-a180-69ef05ecff21)
>

File: src/components/Navbar.js [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+          <NavLink to="/categories" activeClassName="active">
             CATEGORIES
           </NavLink>
         </li>
       </ul>
     </nav>
+    <div className="user-avatar">
+      <img src={avatar} alt="User Avatar" />
+      <span className="username">{username}</span>
+    </div>
```
Comment (Johnadibe) [inline src/components/Navbar.js]: > - [ ] Please ensure that the avatar image is adjusted to adhere to the provided design. 👍
> 
> Provided design:
> 
> qa
> 
> Your design:
> 
> ot
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @mumbereh👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitHub Flow correctly 👍🏽
✅ Linter checks passed✔️
✅ Project is documented professionally✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @mumbereh  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Graciano1997** [comment]
@mumbereh Great Job and Happy coding

**Pull request merged**